### ISSUE TITLE
Issues uncovered during initial accessibility feature testing

### ISSUE THREAD

**Issue by blundellj** [Smithsonian/dpo-voyager#105]
### Edge and Safari on Mac
Pressing “exc” while in full screen with a menu/tool/reader up always exits full screen first, then exits the menu/tool/reader on subsequent presses. Is this desired? It feels backwards to me (exiting full screen should happen last, after all menus are exited).

Exiting the tool menus by selecting there icon seemed strange at first, should we add a separate “x” button?

The Tools>Lights>Intensity slider is slow to move, it should move faster when the arrow buttons are pressed (my mac keyboard has no pageup or pagedown keys)

The Tools>Lights> Color picker does not seem to work and escaping the color picker closes out the entire lighting tool, instead of just the color picker. It seems like when you open the color picker you should be locked into it until you leave to tool.

Set Language picker selection highlighting (outline) does not display correctly. If Spanish is active and you want to select English, it is not clear what you are selecting. It’s not that clear even when trying to switch from English to Spanish

Unable to scroll down articles that are included in a tour

### Safari on Mac
Article Reader does not capture tab navigation, i.e. if you press tab enough you can navigate through the left hand menu (this is not the case on Edge)

### Edge on Mac
Tools>Lights>Intensity Slider. Having to click “space” on the Intensity bar to activate it for manipulation with the arrow keys feels like an extra button press, also the is no indication that anything has changed (that you are now able to manipulate the Intensity slider) after clicking “space” on it. It seems that if you are over the tool then you should be able to slide it

**Comment by gjcope** [comment]
> ### Edge and Safari on Mac
> Pressing “exc” while in full screen with a menu/tool/reader up always exits full screen first, then exits the menu/tool/reader on subsequent presses. Is this desired? It feels backwards to me (exiting full screen should happen last, after all menus are exited).

Unfortunately when you hit "Esc" while in fullscreen, it's handled by the browser it doesn't generate a key event that we can intercept. We may still be able to stop the default behavior but it would be good to find out what expected behavior is here.

> 
> Exiting the tool menus by selecting there icon seemed strange at first, should we add a separate “x” button?

I'm not sure what this means. Do you have an example?

> 
> The Tools>Lights>Intensity slider is slow to move, it should move faster when the arrow buttons are pressed (my mac keyboard has no pageup or pagedown keys)

This is the WCAG pattern for slider control (https://www.w3.org/WAI/ARIA/apg/patterns/slider/). If we speed up arrow movement, fine tuning would no longer be possible. What speed are you proposing?

> 
> The Tools>Lights> Color picker does not seem to work and escaping the color picker closes out the entire lighting tool, instead of just the color picker. It seems like when you open the color picker you should be locked into it until you leave to tool.

Unfortunately with the current color picker implementation, they are no longer tab focusable (https://bugs.webkit.org/show_bug.cgi?id=194756). This is captured in another issue.

> 
> Set Language picker selection highlighting (outline) does not display correctly. If Spanish is active and you want to select English, it is not clear what you are selecting. It’s not that clear even when trying to switch from English to Spanish

Is this still an issue? I am unable to reproduce it.

> 
> Unable to scroll down articles that are included in a tour

I'm not able to reproduce this. Scrolling with arrow keys seems to work fine.

> 
> ### Safari on Mac
> Article Reader does not capture tab navigation, i.e. if you press tab enough you can navigate through the left hand menu (this is not the case on Edge)

This seems to be a Safari limitation. You have to 'turn on' full tab accessibility in the settings, or use "Option + Tab" to navigate. When I use "Option + Tab" it seems to work as expected.

> 
> ### Edge on Mac
> Tools>Lights>Intensity Slider. Having to click “space” on the Intensity bar to activate it for manipulation with the arrow keys feels like an extra button press, also the is no indication that anything has changed (that you are now able to manipulate the Intensity slider) after clicking “space” on it. It seems that if you are over the tool then you should be able to slide it

Is this still the case? It seems to be working as expected on the current version of Edge.

**Comment by blundellj** [comment]
Most issues addressed, opening new ticket for remaining issue with more context

Pressing “exc” while in full screen
- still occurs

Exiting the tool menus
- I don't know what I was talking about... plz disregard

The Tools>Lights>Intensity slider is slow to move
- Couldn't test because of separate issue

Set Language picker selection
- Couldn't reproduce

Unable to scroll
- Sill issue, see https://3d.si.edu/object/3d/red-starfleet-uniform-worn-nichelle-nichols-lt-uhura-star-trek:07a4621e-4c5c-4e3d-b376-12eb2c3c3c58
